schema.sql

```diff
@@ -71,3 +72,11 @@ CREATE TABLE visits (
     PRIMARY KEY (animals_id, vets_id, date_of_visit)
 );
 
+-- Second week - Performance Audit
+
+CREATE INDEX animal_id_asc ON visits (animal_id);
+
+CREATE INDEX mail_asc ON owners (email);
```

Comment:
### [OPTIONAL]

This is a nice-to-have a suggestion as it won't improve the query that was reported as problematic in the requirements for this project.

As email addresses are not case sensitive, we might need to look for the emails in a case insensitive way. There is an option to reflect that in the index by using:

 `CREATE INDEX ON owners ((lower(email)));`